packages/ui/src/GlobalStyle/global.style.ts

```diff
@@ -5,13 +5,14 @@ export const GlobalStyle = createGlobalStyle`
     * {
         box-sizing: border-box;
     }
-    html {
-        font-size: 62.5%;
-    }
     body {
-        font-size: 1.6rem;
+        font-size: 1rem;
         font-family: "Inter", -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
         margin: 0;
         padding: 0;
     }
+    button {
+        font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+      Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    }
```

Comment:
Are you sure we need this?

Reply:
If we don't declare the `font-family` for button as well, the user agent stylesheet overrides the font we have in the body because the specificity is bigger. Do you have another solution?

Reply:
Perhaps try see if `inherit` will do the job?

Reply:
Yep, very true `inherit` will do the job! My bad.